packages/tracing/src/hubextensions.ts

```diff
@@ -33,12 +39,9 @@ function traceHeaders(this: Hub): { [key: string]: string } {
  *
  * @returns The given transaction with its `sampled` value set
  */
-function sample<T extends Transaction>(hub: Hub, transaction: T, samplingContext: SamplingContext): T {
-  const client = hub.getClient();
-  const options = (client && client.getOptions()) || {};
-
-  // nothing to do if there's no client or if tracing is disabled
-  if (!client || !hasTracingEnabled(options)) {
+function sample<T extends Transaction>(transaction: T, options: Options, samplingContext: SamplingContext): T {
```

Comment:
This should probably be a method on `Transaction`, but didn't want to make that change now to keep this patch small.

I did however change the argument order to have transaction as the first argument, since this function operates (mutates!) the transaction, and removed the dependency on the hub as we only need to know about the client options.

Reply:
I think that the `options` should go last though, but it's just a nitpick :P

Reply:
yeah agree, I'll leave that for later so we don't have to go change more lines on the call sites.